node-app/templates/deployment.yaml

```diff
@@ -21,6 +21,13 @@ spec:
       serviceAccountName: {{ include "node-app.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
this needs to be moved to a job and use the helm pre-install hook to run the job